defaultclient.go

```diff
@@ -242,3 +252,73 @@ func (client *DefaultClient) HealthCheck() bool {
 	}
 	return true
 }
+
+// ValidateAudienceScope validate audience and scope of user access token
+func (client *DefaultClient) ValidateAudienceScope(claims *JWTClaims, scope string) error {
+	if baseURI == "" {
+		err := client.getClientInformation(claims.Namespace)
+		if err != nil {
+			return err
+		}
+	}
+
+	if scope == "" {
+		scope = defaultScope
+	}
+
+	if scope != claims.Scope {
+		return errors.New("scope isn't valid")
+	}
+
+	var isValid = false
+	for _, reqAud := range claims.Audience {
+		if reqAud == baseURI {
+			isValid = true
+		}
+	}
+
+	if !isValid {
+		return errors.New("audience isn't valid")
+	}
+
+	return nil
+}
+
+func (client *DefaultClient) getClientInformation(namespace string) (err error) {
```

Comment:
you need to mention that this needs client credentials login, because it uses client access token. At least in the docs

Reply:
do you mean that docs is Readme?

Reply:
already put a notes in docs